docs/index.rst

```diff
@@ -1,19 +1,34 @@
 ..
-  Copyright 2020 Intel Corporation
+  Copyright 2020�2022 Intel Corporation
 
 .. _onemkl:
 
 *****************
 oneMKL Interfaces
 *****************
 
-Introduction
-============
+Contents
+========
 
-oneMKL Interfaces is an open-source implementation of oneMKL Data Parallel C++ (DPC++) interfaces accordingly to `oneMKL specification <https://spec.oneapi.com/versions/latest/elements/oneMKL/source/index.html>`_ that can work with multiple devices (backends) using device specific libraries underneath
+.. toctree::
+   :caption: About
+   :maxdepth: 2
+
+   introduction.rst
+
+.. toctree::
+   :caption: Get Started
+   :maxdepth: 2
 
+   selecting_a_compiler.rst
+   build_setup.rst
+   build_setup_with_hipsycl.rst
+   building_with_conan.rst
+   building_with_cmake.rst
+   project_cleanup.rst
```

Comment:
Do you think we can't combine these files into one .rst file? It makes more sense to me to see all this information together for how to build projects instead of having many files.

Reply:
Sure. I'll incorporate the ones underneath "Selecting a Compiler" (which I believe should remain a separate topic) into a single "Building the Project" topic if there are no objections.

Reply:
Looks good to me. Thank you!

Reply:
This is done. @mkrainiuk: Todd and I will get together to submit the special GitHub Pages pull request for publishing. Can you then merge both PRs "simultaneously"? Thanks! (Then we should check some links in the readme.)